features/FEATURE_PSA/supporting_psa_in_mbed-os.md

```diff
@@ -177,14 +154,19 @@ The following example shows a PSA enabled dual-core target, `PSoC64`,
         "reset_method": "default",
         "post_binary_hook": {
             "function": "PSOC6Code.sign_image"
-        }
-    },
+        },
+        "tfm_target_name": "psoc64_1m",
+        "tfm_bootloader_supported": false,
+        "tfm_default_toolchain": "ARMCLANG",
+        "tfm_supported_toolchains": ["ARMCLANG", "GNUARM"],
+        "tfm_delivery_dir": "TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CPROTO_064_SB"
```

Comment:
Could the deliver dir be computed automatically?

Reply:
Not without searching through Mbed OS tree using target name.

Reply:
Too bad. Maybe when we have a better build system we can put this in the right place automatically.